src/ai/backend/manager/scheduler/dispatcher.py

```diff
@@ -23,6 +23,8 @@
 import sqlalchemy as sa
 from sqlalchemy.ext.asyncio import AsyncConnection as SAConnection
 from sqlalchemy.sql.expression import true
+from sqlalchemy.orm.exc import NoResultFound
+from sqlalchemy.orm.exc import MultipleResultsFound
```

Comment:
Merge into one line because the import source is same.
```suggestion
from sqlalchemy.orm.exc import MultipleResultsFound, NoResultsFound
```